Kakaotalk/Model/CustomObject/User.swift

```diff
@@ -2,7 +2,7 @@
 //  User.swift
 //  Kakaotalk
 //
-//  Created by 강성찬 on 2023-04-25.
+//  Created by 강성찬 on 2023-05-01.
```

Comment:
These dates should be `created by date` and do not need to update them every day.

Reply:
hi